addons/fluxcd/readme.md

```diff
@@ -16,3 +16,38 @@ Enable fluxcd addon to use these X-definitions
 git repo / helm repo / S3 compatible bucket.
 
 - [kustomize](https://kubevela.io/docs/end-user/components/kustomize) helps to deploy a kustomize style artifact.
+
+## Note
+- In this Fluxcd Addon, there are five controllers to be installed by default
+
+```shell
+USED BY CONTROLLER              DEFINITION NAME                         DEFINITION TYPE           DEFINITION DESCRIPTION
+helm-controller                config-helm-repository                  ComponentDefinition       Config information to authenticate helm chart repository
+helm-controller                helm                                    ComponentDefinition       helm release is a group of K8s resources from either git
+                                                                                                 repository or helm repo
+kustomize-controller           kustomize                               ComponentDefinition       kustomize can fetching, building, updating and applying
+                                                                                                 Kustomize manifests from git repo.
+kustomize-controller           kustomize-json-patch                    TraitDefinition           A list of JSON6902 patch to selected target
+kustomize-controller           kustomize-patch                         TraitDefinition           A list of StrategicMerge or JSON6902 patch to selected
+                                                                                                 target
+kustomize-controller           kustomize-strategy-merge                TraitDefinition           A list of strategic merge to kustomize config
+```
+- Source controller
+  - The source-controller is a Kubernetes operator, specialised in artifacts acquisition from external sources such as Git, Helm repositories and S3 buckets. The source-controller implements the source.toolkit.fluxcd.io API and is a core component of the GitOps toolkit.
+  - ![overview](../../addons/fluxcd/source-controller-overview.png)
```

Comment:
you don't need to copy the image into the addon folder, you can direct use the link from their website.